.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "browser": true,
+    "commonjs": true,
```

Comment:
The source code no longer uses common js modules since it's all TypeScript, and TypeScript has its own module system
```suggestion
```